contracts/Vault.vy

```diff
@@ -849,7 +849,11 @@ def maxAvailableShares() -> uint256:
 
 
 @external
-def withdraw(_shares: uint256 = MAX_UINT256, recipient: address = msg.sender) -> uint256:
+def withdraw(
+    maxShares: uint256 = MAX_UINT256,
+    recipient: address = msg.sender,
+    maxLoss: uint256 = 0,  # BPS
```

Comment:
a default of zero means that if there is a dust loss from rounding this will revert. Suggest setting it to 1. (0.01%)

Reply:
```suggestion
    maxLoss: uint256 = 1,  # 0.01% [BPS]
```